clj-refactor.el

```diff
@@ -1282,31 +1284,34 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-move-form"
         (ido-find-file)
         (setq ns (cljr--current-namespace)
               names (cljr--name-of-defns forms)
-              target-ns-alias (-some->>
-                               (--filter (s-matches-p (format target-ns-regexp-template ns) it) requires)
-                               (car)
-                               (s-slice-at ":as")
-                               (-last-item)
-                               (replace-regexp-in-string (format target-ns-alias-template ns) "\\1")
-                               (s-trim)))
+	      target-ns-alias (when-let ((filtered-require (seq-find
+							    (lambda (it)
+							      (s-matches-p (format target-ns-regexp-template ns) it))
+							    requires)))
+				(thread-last filtered-require
+				  (s-slice-at ":as")
+				  (last)
+				  (car)
+				  (replace-regexp-in-string (format target-ns-alias-template ns) "\\1")
```

Comment:
I would say this part of the thread always returns a string, so `nil` shouldn't be propagated. But this new expression is less flexible than former with `-some->>`, and if someone introduces some new function in the thread, should be more careful with it.